### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by Swaagie**
## Summary

Update `README.md`

## Changelog

Done

## Test Plan

NA

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,4 +27,5 @@
 
 [#1]: https://github.com/godaddy/warehouse.ai-ui/pull/1
 [#2]: https://github.com/godaddy/warehouse.ai-ui/pull/2
-[#5]: https://github.com/godaddy/warehoue.ai-ui/pull/5
```
Comment (msluther) [inline CHANGELOG.md]: 🤣

File: README.md [hunk 2]
```diff
@@ -1,18 +1,71 @@
 # `warehouse.ai-ui`
 
-UI for warehouse.ai
+[![Version npm](https://img.shields.io/npm/v/warehouse.ai-ui.svg?style=flat-square)](https://www.npmjs.com/package/warehouse.ai-ui)
+[![License](https://img.shields.io/npm/l/warehouse.ai-ui.svg?style=flat-square)](https://github.com/godaddy/warehouse.ai-ui/blob/master/LICENSE)
+[![npm Downloads](https://img.shields.io/npm/dm/warehouse.ai-ui.svg?style=flat-square)](https://npmcharts.com/compare/warehouse.ai-ui?minimal=true)
+[![Build Status](https://travis-ci.org/godaddy/warehouse.ai-ui.svg?branch=master)](https://travis-ci.org/godaddy/warehouse.ai-ui)
+[![Dependencies](https://img.shields.io/david/godaddy/warehouse.ai-ui.svg?style=flat-square)](https://github.com/godaddy/warehouse.ai-ui/blob/master/package.json)
+
+UI for [Warehouse.ai]. Visual representation of your builds and status events.
```
Comment (msluther) [inline README.md]: the cli linked to the UI, should this link back?

> For convenience, a [command line interface][wrhs] also exists for [Warehouse.ai].

File: README.md [hunk 3]
```diff
@@ -1,18 +1,71 @@
...
-configuration directory must take the same exact same form. Example config 
-files can be found in [/lib/config](/lib/config). 
+configuration directory must take the same exact same form. Example config
+files can be found in [`/lib/config`](/lib/config).
 Additionally, the configuration files should include configuration information
 for the [warehouse.ai-api-client]. This information should be under the `wrhs`
 key in the config.
 
-```js
+```bash
```
Comment (msluther) [inline README.md]: eh? that certainly looks like javascript in there ;)

File: README.md [hunk 4]
```diff
@@ -1,18 +1,71 @@
...
+npm run build
+```
+
+## API
+
+### GraphQL API
+
+Warehouse UI is backed by a GraphQL API. This API transforms the data from the
+existing warehouse API to a model that is specific to this UI. The GraphQL
+models are based on [warehouse-models] and [warehouse.ai-status-models].
```
Comment (fritzmonkey) [inline README.md]: `warehouse.ai-status-models` isn't defined. Should this be `warehouse-status-models`?

File: README.md [hunk 5]
```diff
@@ -1,18 +1,71 @@
...
+For the client we use [`apollo-client`]. This enables SSR and client caching
+without needing any schema changes.
+
+### Configuration
+
 At a minimum you, must provide that path where which `warehouse.ai-ui` can find
 your configuration. Because this is largely a wrapper around [`slay`], the
-configuration directory must take the same exact same form. Example config 
-files can be found in [/lib/config](/lib/config). 
+configuration directory must take the same exact same form. Example config
```
Comment (fritzmonkey) [inline README.md]: `the same exact same form`. I think we have an extra `same`.

**Pull request merged**